src/tests/Regressions/coreclr/GitHub_76531/test76531.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Needed for mechanical merging of all remaining tests, this particular project may not actually need process isolation -->
```

Comment:
Does the test need the process isolation? If yes, this comment should be deleted. If no, both this comment and the property should be deleted.

Reply:
I need to check that.